src/App.js

```diff
@@ -1,80 +1,10 @@
 import React from 'react';
 import './App.css';
+import Main from './components/main/main';
```

Comment:
Файл компонента должен называться с большой буквы
```suggestion
import Main from './components/main/Main';
```